.changeset/backstage-changelog.js

```diff
@@ -32,7 +32,24 @@ async function getDependencyReleaseLine(changesets, dependenciesUpdated) {
   return ['- Updated dependencies', ...updatedDependenciesList].join('\n');
 }
 
+async function getReleaseLine(changeset, type, options) {
+  const { ignoreUserThanks: users = [] } = options ?? {};
+  const releaseLine = await defaultChangelogFunctions.getReleaseLine(
+    changeset,
+    type,
+    options,
+  );
+  return releaseLine.replace(/Thanks\s(.*)!\s/g, (_, group) => {
+    const links = group
+      .split(', ')
+      .filter(link =>
+        users.every(user => `[@${user}](https://github.com/${user})` !== link),
```

Comment:
This still keeps maintainers if one of them is a maintainer right?

Could you make a `Set` at the top of the function with all of the `[]()` link strings in it, and just excluding those in this filter by checking `.has(link)`? Then if the array turns out to be empty it's still the empty string that's returned, but otherwise at least we keep the non-maintainers only who helped out

Reply:
> This still keeps maintainers if one of them is a maintainer right?

Because it is using _every_, for the filter to return true, the link has to be different from all the ignored users' links (it worked by removing all the maintainer links  when I generated the changeset locally)
But I'm going to refactor it with your suggestion, like how it will make the code easier to read. Thanks for commenting out 🙌🏻

Reply:
Oh 🤦 of course, thanks!

Reply:
Done 🙂 : https://github.com/backstage/backstage/pull/21058/commits/e50d1990f84d152b415eeea77e7df8754f767281